frontend/components/navbar/Navbar.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <div h="10vh" w-full bg-transparent from-transparent to-black bg-gradient-to-t to="50%">
```

Comment:
This was kinda stupid, now if we have website with a lot of text and you scroll it looks bad. So remove it, sorry for inconvenience :3